perf/benchmark/README.md

```diff
@@ -163,34 +159,39 @@ fortioclient   LoadBalancer   xxxx          xxxx       8080:31759/TCP,8079:30495
 export FORTIO_CLIENT_URL=http://$(kubectl get services -n twopods fortioclient -o jsonpath="{.status.loadBalancer.ingress[0].ip}"):8080
 ```
 
-2. Set `PROMETHEUS_URL`: 
+or if you don't have an external IP:
 
 ```bash
-kubectl -n istio-prometheus port-forward $(kubectl -n istio-prometheus get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 & 
+kubectl -n istio-performance port-forward svc/fortioclient 8080:8080 &
```

Comment:
in the examples here isn't it a different namespace than istio-performance?

Reply:
Ups, you are right, I used another namespace for testing and this is simply a copy and paste error. Correct to the `twopods` namespace. Thanks!